### PR TITLE
[RU] Add 'specifics/mnesia' lesson translation

### PR DESCRIPTION
**Pull request by nscyclone**
Could you review this, please, @brain-geek?

### CODE REVIEW

File: ru/lessons/specifics/mnesia.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+lang: ru
+---
+
+Mnesia &mdash; это распределённая система управления базами данных реального времени.
+
+{% include toc.html %}
+
+## Обзор
+
+Mnesia &mdash; это система управления базами данных (СУБД), поставляемая вместе с Erlang Runtime System. Естественно, мы можем использовать её в Elixir. Благодаря *реляционной и гибридной объектной модели данных* Mnesia подходит для распределённых приложений любого масштаба.
```
Comment (brain-geek) [inline ru/lessons/specifics/mnesia.md]: I guess that 'и' is obsolete in the second sentence here.

File: ru/lessons/specifics/mnesia.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+Вопрос выбора конкретной технологии зачастую сбивает с толку. Если вы ответите "да" на любой из следующих вопросов, то это хороший знак, что стоит использовать Mnesia, а не ETS или DETS.
+
+  - Нужен ли мне откат транзакций?
+  - Нужен ли мне простой синтаксис для чтения и записи данных?
+  - Нужно ли мне хранить данные в нескольких нодах?
+  - Нужно ли мне выбирать, где хранить данные &mdash; в RAM или на диске?
+
+## Схема
+
+Mnesia &mdash; это часть ядра Erlang, а не Elixir, поэтому мы обращаемся к ней через двоеточие (см. урок: [Взаимодействие с Erlang](https://elixirschool.com/ru/lessons/advanced/erlang/)):
```
Comment (brain-geek) [inline ru/lessons/specifics/mnesia.md]: I guess that relative link is preferred.

File: ru/lessons/specifics/mnesia.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+```shell
+
+iex> :mnesia.create_schema([node()])
+
+# или если вам по душе стиль Elixir
+
+iex> alias :mnesia, as: Mnesia
+iex> Mnesia.create_schema([node()])
+
```
Comment (brain-geek) [inline ru/lessons/specifics/mnesia.md]: obsolete line

File: ru/lessons/specifics/mnesia.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+```
+
+В течение этого урока мы будем использовать второй способ при работе с Mnesia API. `Mnesia.create_schema/1` инициализирует новую пустую схему и передаёт список нод. В данном случае мы передаём ноду, связанную с нашей IEx-сессией.
```
Comment (brain-geek) [inline ru/lessons/specifics/mnesia.md]: `В течение этого урока` -> `В этом уроке`

File: ru/lessons/specifics/mnesia.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+Interactive Elixir ({{ site.elixir.version }}) - press Ctrl+C to exit (type h() ENTER for help)
+iex(learner@elixirschool.com)> Node.self
+:"learner@elixirschool.com"
+```
+
+Теперь, как вы видите, текущая нода &mdash; это атом с именем `:"learner@elixirschool.com"`. Если мы снова выполним `Mnesia.create_schema([node()])`, то мы увидим ещё одну папку с именем **Mnesia.learner@elixirschool.com**. Цель всего этого довольно проста: ноды в Erlang используются для связи с другими нодами, чтобы разделять информацию и ресурсы. Это работает не только в рамках одной машины, но также через LAN и интернет.
+
+## Запуск Mnesia
+
+Мы закончили базовую настройку нашей базы и теперь можем запустить Mnesia с помощью команды ```Mnesia.start/0```.
```
Comment (brain-geek) [inline ru/lessons/specifics/mnesia.md]: you don't need three "`"s here, one should be enough

File: ru/lessons/specifics/mnesia.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+:ok
+iex> Mnesia.start()
+:ok
+```
+
+Важно помнить, что при использовании Mnesia на двух и более нодах функцию `Mnesia.start/1` надо выполнить на каждой из них.
+
+## Создание таблиц
+
+Для создания таблиц используется функция `Mnesia.create_table/2`. Ниже создадим таблицу `Person` и передадим ключевой список с её схемой.
```
Comment (brain-geek) [inline ru/lessons/specifics/mnesia.md]: `Ниже` - > `Сейчас мы`?

File: ru/lessons/specifics/mnesia.md [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+
+```shell
+iex> Mnesia.create_table(Person, [attributes: [:id, :name, :job]])
+{:atomic, :ok}
+```
+
+Мы объявили колонки, используя атомы `:id`, `:name`, и `:job`. После выполнения `Mnesia.create_table/2` возвращает один из следующих ответов:
+
+ - `{atomic, ok}`, если функция выполнилась успешно;
```
Comment (brain-geek) [inline ru/lessons/specifics/mnesia.md]: missing semicolons?

File: ru/lessons/specifics/mnesia.md [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+ - `{aborted, Reason}` в случае ошибки.
```
Comment (brain-geek) [inline ru/lessons/specifics/mnesia.md]: missing semicolons?

File: ru/lessons/specifics/mnesia.md [hunk 9]
```diff
@@ -0,0 +1,177 @@
...
+
+## "Грязный" способ
+
+Сначала рассмотрим "грязный" способ чтения и записи. Как правило, его не используют, потому что он не гарантирует результат, однако он поможет нам комфортно работать с Mnesia во время обучения. Добавим несколько записей в таблицу **Person**:
```
Comment (brain-geek) [inline ru/lessons/specifics/mnesia.md]: `, однако` -> `. Однако` ?

File: ru/lessons/specifics/mnesia.md [hunk 10]
```diff
@@ -0,0 +1,177 @@
...
+:ok
+
+iex> Mnesia.dirty_write({Person, 2, "Homer Simpson", "Safety Inspector"})
+:ok
+
+iex> Mnesia.dirty_write({Person, 3, "Moe Szyslak", "Bartender"})
+:ok
+```
+
+А для получения записей воспользуемся `Mnesia.dirty_read/1`:
```
Comment (brain-geek) [inline ru/lessons/specifics/mnesia.md]: `A` looks obsolete here

File: ru/lessons/specifics/mnesia.md [hunk 11]
```diff
@@ -0,0 +1,177 @@
...
+...>   Mnesia.write({Person, 6, "Monty Burns", "Businessman"})
+...>   Mnesia.write({Person, 7, "Waylon Smithers", "Executive assistant"})
+...> end
+#Function<20.54118792/0 in :erl_eval.expr/5>
+
+iex> Mnesia.transaction(data_to_write)
+{:atomic, :ok}
+```
+
+Судя по ответу, мы можем быть уверены, что данные были записаны в таблицу `Person`. Чтобы убедиться, запросим эти данные другой транзакцией. Для чтения мы используем `Mnesia.read/1` изнутри анонимной функции.
```
Comment (brain-geek) [inline ru/lessons/specifics/mnesia.md]: `мы можем быть уверены, что` can be safely deleted here/

**Comment by brain-geek** [comment]
Hey, @nscyclone , I've left some feedback.

**Comment by nscyclone** [comment]
Thanks, @brain-geek, I've just updated the translation. Could you check it once more?

**Comment by brain-geek** [comment]
@nscyclone Yes, I haven't found any mistakes. Nice job!

**Pull request merged**